infrastructure/resource_group_level/modules/clamav-aca.bicep

```diff
@@ -0,0 +1,86 @@
+param location string = resourceGroup().location
+param environment string
+param acaClamAvSubnetId string
+param logAnalyticsWorkspaceName string
+
+var managedEnvironmentName = 'tfs-${environment}-clamav-env'
```

Comment:
Can we please follow Azure naming conventions, please refer to `https://github.com/UK-Export-Finance/exip/`